src/app/files/file-list.ts

```diff
@@ -68,7 +68,8 @@ import { FileNavService } from './file-nav.service';
                     (dragstart)="drag(child, $event)"
                     (drop)="drop($event, child)"
                     (dragenter)="onDragItemEnter(child, $event)"
-                    (dragleave)="onDragItemLeave(child, $event)">
+                    (dragleave)="onDragItemLeave(child, $event)"
+                    (keyup.space)="select(child)">
```

Comment:
Processing space key to select - is this a well understood feature or should we show as tooltip?

Reply:
Good point. Usually space key used to select focused item. So, users will figure out that they can use the space key for this purpose. 

Ideally, we should support arrow key instead of tab key for list view control. Once we support the arrow key, we can replace space key with right arrow key in the future, supposing the focused item is shown like "> mydir". The extra ">" indicates that the right arrow key can be used to select the current item.  I will open a new issue about that.